### ISSUE TITLE
Ability to mount other dirs to the task containers

### ISSUE THREAD

**Issue by agentmilindu** [leopardslab/Dunner#40]
Users should be able to mount some directories to the Docker containers. Like, 
```
deploy:
- name: 'node'
  mount:
    - './secrets/.npmrc:~/.npmrc:r'
  command: ['npm', 'install']
- name: 'golang'
  mount:
    - '~/go/src/projectname:~/go/src/projectname:wr'
  command: ['go', 'build']
```

`mount` should be an array of strings, and a string has 3 parts as `host-path:container-path:flags`
flags can be optional

**Comment by ayushjn20** [comment]
I would like to work on it.

**Comment by ayushjn20** [comment]
This issues can be closed with reference to #45.

**Comment by agentmilindu** [comment]
Fixed by #45
